### ISSUE TITLE
Keyword parameter names need escaping

### ISSUE THREAD

**Issue by jskeet** [googleapis/gapic-generator-csharp#165]
I've just regenerated Error Reporting, and we've ended up with:

```csharp
public virtual ReportErrorEventResponse ReportErrorEvent(string projectName, ReportedErrorEvent event, gaxgrpc::CallSettings callSettings = null) =>
```

`event` isn't valid as a parameter name, but `@event` is. Note that this isn't just in the method declaration, but in the use too.
Currently working around with a sed script.

**Comment by chrisdunelm** [comment]
Ah yes, keywords are not handled properly as variable names.
Shouldn't be too much problem to fix.

**Comment by chrisdunelm** [comment]
Fixed by #168
